docs/certification.md

```diff
@@ -0,0 +1,83 @@
+#### About  
+Here is writte, how to create self-signed certificates and Certification Authoroty  
+
+##### Generating a Certificate Signing Request (CSR)
+Create the CSR, do not leave passhprase empty for security reasons  
+IMPORTANT!!!   
+Make sure to store the passphrased used.
+
+
+```openssl genrsa -des3 -out server.key 2048```
+
+Create the insecure key, the one without a passphrase, and shuffle the key names
+
+```openssl rsa -in server.key -out server.key.insecure```
+
+```mv server.key server.key.secure```
+```mv server.key.insecure server.key```  
+
+##### The insecure key is now named server.key, and you can use this file to generate the CSR without passphrase.  
+
+Create CSR
+
+```openssl req -new -key server.key -out server.csr```  
+
+Create selfsigned certificate
+
+```openssl x509 -req -days 365 -in server.csr -signkey server.key -out server.crt```  
+
+Install the key file server.key and certificate file server.crt
+
+```sudo cp server.crt /etc/ssl/certs```  
+```sudo cp server.key /etc/ssl/private```
+
+##### Certification Authority  
+
+Create the directories to hold the CA certificate and related files
+
+```sudo mkdir /etc/ssl/CA```  
+```sudo mkdir /etc/ssl/newcerts```  
+
+The CA needs a few additional files to operate, one to keep track of the last serial number used by the CA, each certificate must have a unique serial number, and another file to record which certificates have been issued:
+
+```sudo sh -c "echo '01' > /etc/ssl/CA/serial"```  
+```sudo touch /etc/ssl/CA/index.txt```  
+
+The third file is a CA configuration file. Though not strictly necessary, it is very convenient when issuing multiple certificates. Edit /etc/ssl/openssl.cnf, and in the [ CA_default ] change:  
+
+`
+dir             = /etc/ssl              # Where everything is kept  
+database        = $dir/CA/index.txt     # database index file.  
+certificate     = $dir/certs/cacert.pem # The CA certificate  
+serial          = $dir/CA/serial        # The current serial number  
+private_key     = $dir/private/cakey.pem# The private key  
```

Comment:
```suggestion
```
dir &nbsp;&nbsp;&nbsp;&nbsp; = /etc/ssl &nbsp;&nbsp;&nbsp;&nbsp; # Where everything is kept  
database &nbsp;&nbsp;&nbsp;&nbsp; = $dir/CA/index.txt &nbsp;&nbsp;&nbsp;&nbsp; # database index file.  
certificate &nbsp;&nbsp;&nbsp;&nbsp; = $dir/certs/cacert.pem &nbsp;&nbsp;&nbsp;&nbsp; # The CA certificate  
serial &nbsp;&nbsp;&nbsp;&nbsp; = $dir/CA/serial &nbsp;&nbsp;&nbsp;&nbsp; # The current serial number  
private_key &nbsp;&nbsp;&nbsp;&nbsp; = $dir/private/cakey.pem &nbsp;&nbsp;&nbsp;&nbsp; # The private key  
```